### PR TITLE
Update assign-licenses-to-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed typo

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3790

File: microsoft-365/admin/manage/assign-licenses-to-users.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ When you use the **Active users** page to assign licenses, you assign users lice
 
 ## Assign a license to a guest user
 
-You can invite guest users to collaborate with your organization in Azure Active directory admin center. To learn about guess users, see [What is guest user access in Azure Active Directory B2B?](https://docs.microsoft.com/azure/active-directory/external-identities/what-is-b2b) If you don't have any guest users, see [Quickstart: Add guest users to your directory in the Azure portal](https://docs.microsoft.com/azure/active-directory/external-identities/b2b-quickstart-add-guest-users-portal).
+You can invite guest users to collaborate with your organization in Azure Active directory admin center. To learn about guest users, see [What is guest user access in Azure Active Directory B2B?](https://docs.microsoft.com/azure/active-directory/external-identities/what-is-b2b) If you don't have any guest users, see [Quickstart: Add guest users to your directory in the Azure portal](https://docs.microsoft.com/azure/active-directory/external-identities/b2b-quickstart-add-guest-users-portal).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/assign-licenses-to-users.md]:
```suggestion
You can invite guest users to collaborate with your organization in the Azure Active Directory admin center. To learn about guest users, see [What is guest user access in Azure Active Directory B2B?](https://docs.microsoft.com/azure/active-directory/external-identities/what-is-b2b). If you don't have any guest users, see [Quickstart: Add guest users to your directory in the Azure portal](https://docs.microsoft.com/azure/active-directory/external-identities/b2b-quickstart-add-guest-users-portal).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review, a few additional edits, please.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Pull request merged**